packages/ui/src/components/va-input/VaInput.vue

```diff
@@ -142,7 +143,9 @@ export default defineComponent({
   inheritAttrs: false,
 
   setup (props, { emit, attrs, slots }) {
-    const input = ref<HTMLInputElement | typeof VaTextarea | undefined>()
+    const input = ref<HTMLInputElement | HTMLTextAreaElement | undefined>()
+
+    const { isFocused } = useFocus(input, emit)
```

Comment:
I think it would be better to export `onFocus` and `onBlur` from `useFocus` and use them in `inputEvents` (combined with `onFocus` and `onBlur` on lines 191 and 196 below)